src/targets/node/nodeAttacherBase.ts

```diff
@@ -21,7 +21,7 @@ const appendVars: readonly string[] = ['NODE_OPTIONS', 'VSCODE_INSPECTOR_OPTIONS
 export abstract class NodeAttacherBase<T extends AnyNodeConfiguration> extends NodeLauncherBase<T> {
   protected async appendEnvironmentVariables(cdp: Cdp.Api, vars: EnvironmentVars) {
     const expression =
-      `typeof process==='undefined'||process.pid===undefined?'process not defined':(()=>{`
+      `typeof process==='undefined'||process.pid===undefined||typeof Deno==='object'?'process not defined':(()=>{`
```

Comment:
We should return something other than `process not defined` here in the Deno case to avoid the retry loop below

Reply:
`process not defined` already breaks out of the retry loop, right?

Reply:
Sorry, I read that code incorrectly. I'll have it return something else.

Reply:
Yea, for some reason in Node (haven't investigated the internals) the process can be undefined for a short time immediately after attachment, so we retry for a little while